lib/tutorials/en_US/routing.md

```diff
@@ -1,61 +1,73 @@
-## Routing
+# Routing
 
-_This tutorial is compatible with hapi v17_
```

Comment:
We need to keep this information as people come with different versions of hapi and need to know which one this applies to.

Reply:
Done